CONTRIBUTING.md

```diff
@@ -154,6 +154,8 @@ To install all development dependencies, in the project's root directory, run
 npm install
 ```
 
+> Please note: the build process assumes Java is installed locally.
```

Comment:
Sorry to nitpick, but did you mean to italicize/embolden this line?
```suggestion
**Please note: the build process assumes Java is installed locally.**
```

Reply:
No worries. Nope, I generally put `Note: ` at the quote level, but that's a personal pref to be sure. Happy to change to make this bold instead 👍🏼